app/models/server/models/FederationRoomEvents.js

```diff
@@ -1,32 +1,33 @@
-import { FederationEventsModel, contextDefinitions, eventTypes } from './FederationEvents';
+import { EventsModel, eventTypes } from './Events';
+import { getContextQuery } from './RoomEvents';
 
-const { type, contextQuery } = contextDefinitions.ROOM;
+const type = 'NOT USED';
```

Comment:
I didn't understand why this exists still.

Reply:
Probably legacy code, I will check